community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/ValueHashJoinImplementationAcceptanceTest.scala

```diff
@@ -51,7 +51,6 @@ class ValueHashJoinAcceptanceTest extends ExecutionEngineFunSuite with NewPlanne
     val result = executeWithAllPlannersAndCompatibilityMode("MATCH (a:A), (b:B) WHERE a.id = b.id RETURN a, b")
 
     // then
-    result.size should equal(11)
     result should use("ValueHashJoin")
```

Comment:
@systay Given the name of this test, do you think there is an assertion missing? (pinging you as the author of the test)